apps/address-service/domains/address/schema/AddressSource.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Generated by `createschema address.AddressSource 'source:Text;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const access = require('@address-service/domains/address/access/AddressSource')
+
+
+const AddressSource = new GQLListSchema('AddressSource', {
+    schemaDoc: 'The source for the address',
+    labelResolver: ({ source }) => source,
+    fields: {
+
+        source: {
+            schemaDoc: 'Source string itself',
```

Comment:
If I understood it correctly…

```suggestion
            schemaDoc: 'Search string, that was used to find an address string, stored in "address" field',
```

Reply:
Yes, almost right! Not address string, but address model (relation).
schemaDoc updated